src/systems/camera-system.js

```diff
@@ -393,11 +388,11 @@ export class CameraSystem {
         this.avatarRig.object3D.updateMatrices();
         setMatrixWorld(this.viewingRig.object3D, this.avatarRig.object3D.matrixWorld);
         if (scene.is("vr-mode")) {
-          this.viewingCamera.object3D.updateMatrices();
-          setMatrixWorld(this.avatarPOV.object3D, this.viewingCamera.object3D.matrixWorld);
+          this.viewingCamera.object3DMap.camera.updateMatrices();
+          setMatrixWorld(this.avatarPOV.object3D, this.viewingCamera.object3DMap.camera.matrixWorld);
         } else {
           this.avatarPOV.object3D.updateMatrices();
-          setMatrixWorld(this.viewingCamera.object3D, this.avatarPOV.object3D.matrixWorld);
```

Comment:
I forget why there was this complexity around having the `viewingCamera` follow the `avatarPOV` node or vice versa, but I thought it was related to the `setPoseTarget` stuff and how it interacted with the-mode-formerly-known-as-inspect (aka focus mode). I'd just want to double check that focus and object list navigation works in 2D, touchscreen, and in VR.